testing/trino-product-tests/src/main/java/io/trino/tests/product/deltalake/TestDeltaLakeIdentityColumnCompatibility.java

```diff
@@ -52,30 +56,269 @@ public void testIdentityColumn()
                 bucketName,
                 tableDirectory));
         try {
-            String failureMessage = ".* Table .* requires Delta Lake writer version 6 which is not supported";
-            assertQueryFailure(() -> onTrino().executeQuery("COMMENT ON COLUMN delta.default." + tableName + ".b IS 'test column comment'"))
-                    .hasMessageMatching(failureMessage);
-            assertQueryFailure(() -> onTrino().executeQuery("COMMENT ON TABLE delta.default." + tableName + " IS 'test table comment'"))
-                    .hasMessageMatching(failureMessage);
-            assertQueryFailure(() -> onTrino().executeQuery("ALTER TABLE delta.default." + tableName + " ADD COLUMN c INT"))
-                    .hasMessageMatching(failureMessage);
-            assertQueryFailure(() -> onTrino().executeQuery("INSERT INTO delta.default." + tableName + "(a) VALUES (0)"))
-                    .hasMessageMatching(failureMessage);
-            assertQueryFailure(() -> onTrino().executeQuery("DELETE FROM delta.default." + tableName))
-                    .hasMessageMatching(failureMessage);
-            assertQueryFailure(() -> onTrino().executeQuery("MERGE INTO delta.default." + tableName + " t USING delta.default." + tableName + " s " + "ON (t.a = s.a) WHEN MATCHED THEN UPDATE SET b = 1"))
-                    .hasMessageMatching(failureMessage);
+            onTrino().executeQuery("COMMENT ON COLUMN delta.default." + tableName + ".b IS 'test column comment'");
+            onTrino().executeQuery("COMMENT ON TABLE delta.default." + tableName + " IS 'test table comment'");
+            onTrino().executeQuery("ALTER TABLE delta.default." + tableName + " ADD COLUMN c INT");
+            // Don't execute other column operations because column mapping mode 'none' doesn't support it. See other test methods in this class.
 
             assertThat((String) onDelta().executeQuery("SHOW CREATE TABLE default." + tableName).getOnlyValue())
                     .contains("b BIGINT GENERATED ALWAYS AS IDENTITY");
-            onDelta().executeQuery("INSERT INTO default." + tableName + " (a) VALUES (0)");
+            onDelta().executeQuery("INSERT INTO default." + tableName + " (a, c) VALUES (0, 2)");
 
-            QueryAssert.Row expected = row(0, 1);
+            QueryAssert.Row expected = row(0, 1, 2);
             assertThat(onTrino().executeQuery("SELECT * FROM delta.default." + tableName)).containsOnly(expected);
             assertThat(onDelta().executeQuery("SELECT * FROM default." + tableName)).containsOnly(expected);
         }
         finally {
             dropDeltaTableWithRetry("default." + tableName);
         }
     }
+
+    @Test(groups = {DELTA_LAKE_DATABRICKS, DELTA_LAKE_EXCLUDE_73, DELTA_LAKE_EXCLUDE_91, DELTA_LAKE_EXCLUDE_104, DELTA_LAKE_EXCLUDE_113, PROFILE_SPECIFIC_TESTS})
+    @Flaky(issue = DATABRICKS_COMMUNICATION_FAILURE_ISSUE, match = DATABRICKS_COMMUNICATION_FAILURE_MATCH)
+    public void testIdentityColumnFeature()
```

Comment:
the other method is called `testIdentityColumn`.
"Feature" suffix doesn't add any information, so it feels like "testIdentityColumn2".

can we name this method differently?

Reply:
The "Feature" comes from [table **features**](https://github.com/delta-io/delta/blob/master/PROTOCOL.md#table-features) in Delta. Renamed to `testIdentityColumnTableFeature`